dhis2_clone.py

```diff
@@ -372,8 +372,9 @@ def get_db(cfg, args):
         cmd = "ssh %s %s | gzip > %s" % (cfg["hostname_remote"], dump, sql_path)
         run(cmd)
         apps_dir = os.path.join(dir_local, "files", "apps")
-        d2_docker_cmd = "d2-docker create data {} --sql={} --apps-dir {}".format(
-            get_local_docker_image(cfg, args, "stop"), sql_path, apps_dir
+        documents_dir = os.path.join(dir_local, "files", "document")
+        d2_docker_cmd = "d2-docker create data {} --sql={} --apps-dir {} --documents-dir {}".format(
+            get_local_docker_image(cfg, args, "stop"), sql_path, apps_dir, documents_dir
```

Comment:
A caveat: folder `files/document` is not created until a first resource is uploaded, so it may not exist and d2-docker will complain about it. Maybe add a `os.path.isdir` check?

Reply:
Still shouldn't be d2-docker the one to validate if the folder exist?

``os.path.join`` should not fail if the folder does not exist.

Reply:
I think `os.path.join` does not care about the filesystem at all. 

Yes, I can add that check to d2-docker, it's only that it felt safer to keep failing loudly when you pass something wrong (ie. it will catch a typo in the path)

Reply:
Got your point, adding it here 👍🏻